modules/user-agent-overview.adoc

```diff
@@ -23,11 +23,11 @@ For example, when:
 * `<command>` = `oc`
 * `<version>` = The client version. For example, `v4.3.0`. Requests made against the Kubernetes
 API at `/api` receive the Kubernetes version, while requests made against the
-{product-title} API at `/oapi` receive the {product-title} version (as specified
+{product-title} API at `/apis/build.openshift.io` receive the {product-title} version (as specified
```

Comment:
Not sure how this is done outside of `/oapi` - this content may no longer be relevant.

cc @sttts